src/auth/init-authentication.ts

```diff
@@ -46,6 +46,9 @@ async function signUpHandler(): Promise<void> {
 
       window.showInformationMessage(authMessage.signedUp)
       ext.outputChannel.appendLine(authMessage.signedUp)
+
+      await iamHelpers.setupInitialProject()
```

Comment:
While this is a good place for this, I feel like we could do even better by putting this somewhere closer to `get active project` flow.

Because if user signed up BEFORE this PR is merged, the default project won't be created for them.

Maybe we could do something like:
```ts
  async requireActiveProjectId(): Promise<string> {
    const userConfig = await this.getUserConfig()
    if (userConfig && userConfig.activeProjectId) {
      return userConfig.activeProjectId
    }

    await iamHelpers.createDefaultProject()

    const projects = await iamState.listProjects()
    const activeProjectId = projects[0].projectId

    await this.setUserConfig({ activeProjectId })

    return activeProjectId
  }
```

And here (after sign up), we'll leave this function too:
```suggestion
      await iamHelpers.createDefaultProject()
```

And the function itself:
```ts
let promise: Promise<void> | undefined

export async function createDefaultProject(): Promise<void> {
  if (!promise) {
    promise = (async () => {
      if ((await iamState.listProjects()).length > 0) return

      await window.withProgress({
        location: ProgressLocation.Notification,
        title: projectMessage.creatingDefaultProject
      }, () => iamClient.createProject(
        { name: 'Default Project' },
        { consoleAuthToken: await authHelper.getConsoleAuthToken() }
      ))

      await iamState.clear()
      ext.explorerTree.refresh()
    })()
  }

  return promise
}
```

I'm storing a promise so that if this function gets called twice accidentally, we won't create two default projects (avoiding race condition)

Also let's not use `createProjectProcess` here because I feel like this is a special case and we shouldn't make that function more complex than it already is

Reply:
Or maybe we should event call this in `listProjects()` method.. hard to decide :D